chapters/en/chapter10/4.mdx

```diff
@@ -0,0 +1,343 @@
+|----------------|-------------|---------------------------------------|------------------------|----------------------|---------------------------|---------------------------------------------------|
+| LibriSpeech    | 960         | Audiobooks                            | Narrated               | ❌                    | ❌                         | Academic benchmarks                               |
+| Common Voice 9 | 2224        | Wikipedia text + crowd-sourced speech | Narrated               | ✅                    | ✅                         | Day-to-day speech                                 |
+| TEDLIUM        | 452         | TED talks                             | Narrated               | ❌                    | ❌                         | Technical scientific, political and social topics |
+| Voxpopuli      | 543         | European Parliament recordings        | Spontaneous            | ❌                    | ❌                         | Non-native English speakers                       |
+| GigaSpeech     | 10000       | Audiobook, podcast, youtube           | Narrated & spontaneous | ❌                    | ✅                         | Robustness over multiple domains                  |
+| SPGISpeech     | 5000        | Financial meetings                    | Narrated & spontaneous | ✅                    | ✅                         | Fully formatted transcriptions                    |
+
+
+Let’s take an example where we want to train a speech recognition system to transcribe speech on the topic of machine learning (highly relevant!). Our speech is going to be narrated (scripted), and we’re not worried about punctuation or casing on our text output. From our reference table, it looks like TED-LIUM is a good choice of dataset!
+
+[//]: # On review, this sub-section might be too involved. Happy to omit it and dive straight into how to load a dataset.
+
+### How to load an audio dataset
+Before we can handle any audio datasets, we need to make sure we have the right dependencies installed. Specifically, we'll need the "Audio" feature from datasets: https://huggingface.co/docs/datasets/installation#audio. This should take care of the main Python packages we need to read audio files from byte form and convert them to arrays.
+
+Right! Now we're ready to go ahead and download our data. For this tutorial, we'll use the [_LibriSpeech ASR_](https://huggingface.co/datasets/librispeech_asr) corpus. LibriSpeech is one of the most popular datasets for benchmarking speech recognition systems in both academia and industry. It consists of approximately 1000 hours of narrated audiobooks collected from the (LibriVox)[https://librivox.org/] project.
+
+
+Let’s take a look at the all-time leaderboard on Papers with Code to get a feel for numbers: https://paperswithcode.com/sota/speech-recognition-on-librispeech-test-clean
```

Comment:
I suggest using a screenshot here (with link), and also point the reader to our own leaderboards (derived from Hub models): https://huggingface.co/spaces/autoevaluate/leaderboards?dataset=librispeech_asr